README.md

```diff
@@ -1,20 +1,13 @@
-iojs/iojs
+iojs
 =========
 
-[![dockeri.co](http://dockeri.co/image/iojs/iojs)](https://registry.hub.docker.com/u/iojs/iojs/)
+[![dockeri.co](http://dockeri.co/image/_/iojs)](https://registry.hub.docker.com/_/iojs/)
```

Comment:
> this is a bit premature isn't it? afaik it's not safe to assume this will become the official repo

Yes, this wasn't intended to be merged until the repo actually became official.

> from the look of the discussion they are talking about maintaining their own official one separate to this

Not really, this repo will still be the definitive source for the image. The process is a bit different for official image, I had posted links to docs in #1